protos/workflow/workflow.proto

```diff
@@ -39,7 +40,7 @@ service WorkflowSvc {
             get: "/v1/workflows/{id}/events"
         };
     };
-    rpc GetWorkflowContexts(WorkflowContextRequest) returns (WorkflowContextList) {}
+    rpc GetWorkflowContexts(WorkflowContextRequest) returns (stream WorkflowContext) {}
```

Comment:
Is changing to a stream backwards compatible?

Reply:
No, I don't think so, but Since this function is only called from worker side so I have changed this only instead of having a new gRPC API. Is that ok?